build.gradle

```diff
@@ -284,6 +284,7 @@ task writeCommonProperties(type: WriteProperties) {
 
 processResources {
     from(writeProperties)
+    from(writeCommonProperties)
```

Comment:
Missed including this when task was added. App name wasn't getting updated with new version. 😅